docs/readthedocs/source/doc/PPML/Overview/ppml.md

```diff
@@ -222,39 +222,177 @@ The result should look like this:
+kubectl config view --flatten --minify > /YOUR_DIR/kubeconfig
 ```
-##### 2.2.3.3  Stop Distributed Big Data and ML Platform
-
-Firstly stop the training:
+Create k8s secret, the secret created `YOUR_SECRET` should be the same as the password you specified in section 2.2.3.1:
 
 ```bash
-./stop-distributed-standalone-spark.sh
+kubectl create secret generic spark-secret --from-literal secret=YOUR_SECRET
 ```
 
-Then stop the service:
+##### 2.2.3.3  Start the client container
+
+Configure the environment variables in the following script before running it. Check [Bigdl ppml SGX related configurations](https://github.com/intel-analytics/BigDL/tree/main/ppml/trusted-big-data-ml/python/docker-graphene#1-bigdl-ppml-sgx-related-configurations) for detailed memory configurations. 
 
 ```bash
-./undeploy-distributed-standalone-spark.sh
-```
+export K8S_MASTER=k8s://$( sudo kubectl cluster-info | grep 'https.*' -o -m 1 )
+echo The k8s master is $K8S_MASTER .
+export ENCLAVE_KEY=/YOUR_DIR/enclave-key.pem
+export DATA_PATH=/YOUR_DIR/data
+export KEYS_PATH=/YOUR_DIR/keys
+export SECURE_PASSWORD_PATH=/YOUR_DIR/password
+export KUBECONFIG_PATH=/YOUR_DIR/kubeconfig
+export LOCAL_IP=$LOCAL_IP
+export DOCKER_IMAGE=intelanalytics/bigdl-ppml-trusted-big-data-ml-python-graphene:2.1.0-SNAPSHOT
+sudo docker run -itd \
+    --privileged \
+    --net=host \
+    --name=spark-local-k8s-client \
+    --cpuset-cpus="0-4" \
+    --oom-kill-disable \
+    --device=/dev/sgx/enclave \
+    --device=/dev/sgx/provision \
+    -v /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket \
+    -v $ENCLAVE_KEY:/graphene/Pal/src/host/Linux-SGX/signer/enclave-key.pem \
+    -v $DATA_PATH:/ppml/trusted-big-data-ml/work/data \
+    -v $KEYS_PATH:/ppml/trusted-big-data-ml/work/keys \
+    -v $SECURE_PASSWORD_PATH:/ppml/trusted-big-data-ml/work/password \
+    -v $KUBECONFIG_PATH:/root/.kube/config \
+    -e RUNTIME_SPARK_MASTER=$K8S_MASTER \
+    -e RUNTIME_K8S_SERVICE_ACCOUNT=spark \
+    -e RUNTIME_K8S_SPARK_IMAGE=$DOCKER_IMAGE \
+    -e RUNTIME_DRIVER_HOST=$LOCAL_IP \
+    -e RUNTIME_DRIVER_PORT=54321 \
+    -e RUNTIME_DRIVER_CORES=1 \
+    -e RUNTIME_EXECUTOR_INSTANCES=1 \
+    -e RUNTIME_EXECUTOR_CORES=8 \
+    -e RUNTIME_EXECUTOR_MEMORY=1g \
+    -e RUNTIME_TOTAL_EXECUTOR_CORES=4 \
+    -e RUNTIME_DRIVER_CORES=4 \
+    -e RUNTIME_DRIVER_MEMORY=1g \
+    -e SGX_DRIVER_MEM=32g \
+    -e SGX_DRIVER_JVM_MEM=8g \
+    -e SGX_EXECUTOR_MEM=32g \
+    -e SGX_EXECUTOR_JVM_MEM=12g \
+    -e SGX_ENABLED=true \
+    -e SGX_LOG_LEVEL=error \
+    -e SPARK_MODE=client \
+    -e LOCAL_IP=$LOCAL_IP \
+    $DOCKER_IMAGE bash
+```
+
+##### 2.2.3.4 Init the client and run Spark applications on k8s
+
+##### 2.2.3.4.1 Init the Spark local k8s client
+
+First, run `docker exec -it spark-local-k8s-client bash` to entry the container. Then run the following command to init the Spark local k8s client
+
+```bash
+./init.sh
+```
+
+##### 2.2.3.4.2 Configure spark-executor-template.yaml in the container

```

Comment:
2.2.3.4.2 -> 2

Reply:
update in https://github.com/intel-analytics/BigDL/pull/4515/files#diff-206dc464459fee64731e43f5909c3e6144c71a8f9711eec547deb6ccf7e404e7R316